src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -1166,16 +1181,11 @@ traditionalStatement
         // Return an expression
         |    "return"^ (expression)? SEMI
 
-        // switch/case statement
-        |    "switch"^ LPAREN expression RPAREN LCURLY
-                ( casesGroup )*
-            RCURLY
-
         // exception try-catch block
         |    tryBlock
 
         // throw an exception
-        |    "throw"^ expression SEMI
+        |    throwStatement
```

Comment:
please move this update to separate commit as minor refactoring before functional update.
There will be 3 commits in this PR.

Reply:
Fixed.